lms/static/js/student_profile/views/learner_profile_view.js

```diff
@@ -26,6 +34,17 @@
                     ownProfile: this.options.ownProfile,
                     showFullProfile: this.showFullProfile()
                 }));
+                if (this.showFullProfile() && (this.options.accountSettingsModel.get('accomplishments_shared'))
```

Comment:
Nit Question: is the call to `showFullProfile` redundant since the `accomplishments_shared` field would be set only if the profile is fully shared?